File: sample/src/main/java/com/thefinestartist/finestwebview/sample/WebViewActivity.java
package com.thefinestartist.finestwebview.sample;

import android.os.Bundle;
import android.webkit.WebView;
import androidx.appcompat.app.AppCompatActivity;

/** Created by TheFinestArtist on 3/11/16. */
public class WebViewActivity extends AppCompatActivity {
  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.webview);
    WebView webView = findViewById(R.id.webView);
    webView.getSettings().setJavaScriptEnabled(true);
    webView.loadUrl(
        "http://docs.google.com/gview?embedded=true&url=uet.vnu.edu.vn/~chauttm/e-books/java/Effective.Java.2nd.Edition.May.2008.3000th.Release.pdf");
  }
}

File: sample/src/main/java/com/thefinestartist/finestwebview/sample/WebViewActivity.kt
package com.thefinestartist.finestwebview.sample

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.View
import android.webkit.WebView
import androidx.appcompat.app.AppCompatActivity

/**
 * Created by TheFinestArtist on 3/11/16.
 */
class WebViewActivity : AppCompatActivity() {

    @SuppressLint("SetJavaScriptEnabled")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.webview)

        val url = "http://docs.google.com/gview?embedded=true&url=uet.vnu.edu.vn/~chauttm/e-books/java/Effective.Java.2nd.Edition.May.2008.3000th.Release.pdf"
        val webView = findViewById<View>(R.id.webView) as WebView

        webView.settings.javaScriptEnabled = true
        webView.loadUrl(url)
    }

}